IceCream/Classes/PrivateDatabaseManager.swift

```diff
@@ -143,32 +146,54 @@ final class PrivateDatabaseManager: DatabaseManager {
         
         // if we retry fetch after getting a recoverable error
         var isRetrying = false
-        
+
+        print("^^ fetchChangesInZones started for operation : \(changesOp)")
```

Comment:
What do you think about using Apple's Logger instead of print statements?
https://developer.apple.com/documentation/os/logging

Reply:
@chillpop yeah that is definitely on the TODO list for break work!  we have our own plog stuff but i'm curious if we should move over to Apple's logger for everything